src/api/config.py

```diff
@@ -3,7 +3,7 @@
 
 
 def DB_NAME():
-    DB_NAME = "BD_FINAL"
+    DB_NAME = "postgres"
```

Comment:
Credentials should be passed either by command line or a config file. Never hardcoded.